test/e2e/storage/drivers/csi.go

```diff
@@ -381,8 +381,9 @@ func (g *gcePDCSIDriver) GetDynamicProvisionStorageClass(config *testsuites.PerT
 	if fsType != "" {
 		parameters["csi.storage.k8s.io/fstype"] = fsType
 	}
+	delayedBinding := storagev1.VolumeBindingWaitForFirstConsumer
```

Comment:
Does this mean we always test using delayedBinding for GCE PD driver? Is that OK?

Reply:
I think it's fine because that's what we want to start encourage everyone to use by default.  And we still have other drivers like hostpath that test immediate binding.